lib/index.js

```diff
@@ -58,25 +61,23 @@ module.exports = class ReactBenchmark extends EventEmitter {
     const port = await this.server.start(outputPath)
 
     return new Promise((resolve, reject) => {
-      this.chrome.once('complete', (benchmark) => {
+      this.chrome.once('complete', async (benchmark) => {
         if (!devtools) {
-          this._shutdown()
+          await this._shutdown()
+          resolve(benchmark)
         }
-
-        resolve(benchmark)
       })
 
-      this.chrome.once('error', (err) => {
+      this.chrome.once('error', async (err) => {
         if (!devtools) {
-          this._shutdown()
+          await this._shutdown()
+          reject(err)
         }
-
-        reject(err)
```

Comment:
Won't this cause there to be no error handling when using the DevTools?

Reply:
You are absolutely right, I can not understand how did I've moved this line, committing the change and re-checking my PR one more time